1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -0,0 +1,26 @@
+const MIN_LIMIT = 1
+const MAX_LIMIT = 100
+const FIZZ = 3
+const BUZZ = 5
+
+function FizzBuzz(number: number): string | Error {
```

Comment:
I believe this means that you'll return an error. Are you using it to let the user know that an error might be thrown?

Reply:
yes, that's my intention to throw an error when the number limit is not in range.